infra/terraform/modules/codeowners_file/main.tf

```diff
@@ -20,6 +20,7 @@ locals {
   owners        = { for value in var.repos_map : value.name => "${join(" ", formatlist("@%s", value.owners))} " if length(value.owners) > 0 }
   groups        = { for value in var.repos_map : value.name => "${join(" ", formatlist("@${value.org}/%s", value.groups))} " if length(value.groups) > 0 }
   header        = "# NOTE: This file is automatically generated from values at:\n# https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/terraform/test-org/org/locals.tf\n"
+  footer        = "CODEOWNERS @${var.org}/${var.owner}\n .github/CODEOWNERS @${var.org}/${var.owner}\n docs/CODEOWNERS @${var.org}/${var.owner}\n"
```

Comment:
Could we also add a link for future readers https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-and-branch-protection

Reply:
Good idea - Added a prefix_msg